Library/Homebrew/messages.rb

```diff
@@ -15,8 +15,8 @@ def initialize
     @install_times = []
   end
 
-  def record_caveats(f, caveats)
-    @caveats.push(formula: f.name, caveats: caveats)
+  def record_caveats(software, caveats)
+    @caveats.push(software: software, caveats: caveats)
```

Comment:
```suggestion
  def record_caveats(package, caveats)
    @caveats.push(package: package, caveats: caveats)
```

I've generally seen us use "package" to refer to either a formula or cask. This could be a documentation addition somewhere?

Reply:
That makes perfect sense. 🧠 💤